keanu-project/src/main/java/io/improbable/keanu/vertices/dbl/nonprobabilistic/diff/Infinitesimal.java

```diff
@@ -72,7 +72,7 @@ public Infinitesimal powerTo(double power) {
         return new Infinitesimal(powered);
     }
 
-    public Infinitesimal clone() {
+    public Infinitesimal copy() {
         return new Infinitesimal(cloneInfinitesimals(infinitesimals));
```

Comment:
Maybe change to cloneInfinitesimals -> copyInfinitesimals

Reply:
changed